File: submissions/sm_037_srikanth/week_03/day_2/assignment/taxCalculator.js
var income = 270000;
var savings;
var rebate = 60000;
var tax;
var total_income;

if(income < 500000){
      rebate = (savings/10)*5;
     if(rebate < 50000){
         income = income - rebate;
     }
     else {
         rebate = 50000;
         income = income - rebate;
     }
}
else if((income <= 1000000) && (income >= 500000)){
    rebate = (savings/10)*3;
    if(rebate < 50000){
        income = income - rebate;
    }
    else {
        rebate = 50000;
        income = income - rebate;
    }
}
else {
    rebate = (savings/10);
    if(rebate < 50000){
        income = income - rebate;
    }
    else {
        rebate = 50000;
        income = income - rebate;
    }
}
console.log("Total income after rebate: " + income);
if(income < 250000){
    tax = 0;
    console.log(tax);
}
else if(250000 <= income <= 500000){
    total_income = income - 250000 ;
    tax = total_income/10;
}
else if (500000 < income <= 1000000){
    total_income = income - 500000 ;
    tax = (total_income/10)*2;
}
else{
    total_income = income - 1000000 ;
    tax = (total_income/10)*3;
}
console.log("Total tax: " + tax);

File: submissions/sm_105_ashish/week_13/day_5/tax_calculator.py
Total_Income = int(input("Please Enter your total Income: "))
saving_amount = int(input('Please Enter your savings per year: '))

if saving_amount > Total_Income:
    print('savings can not be greater than total income')

max_cap_deduction = 50000
taxable_amount = 0
total_tax = 0
cap_deduction = 0
if Total_Income < 250000:
    total_tax = 0
elif Total_Income >= 250000 and Total_Income < 500000:
    if saving_amount*0.5 > max_cap_deduction:
        cap_deduction = max_cap_deduction
    else:
        cap_deduction = saving_amount*0.5
    total_tax = (Total_Income - cap_deduction - 250000)*0.1

elif Total_Income >= 500000 and Total_Income < 1000000:
    if saving_amount*0.3 > max_cap_deduction:
        cap_deduction = max_cap_deduction
    else:
        cap_deduction = saving_amount*0.3
    total_tax = (Total_Income - cap_deduction - 500000)*0.2 + 25000

else:
    if saving_amount*0.1 > max_cap_deduction:
        cap_deduction = max_cap_deduction
    else:
        cap_deduction = saving_amount*0.1
    total_tax = (Total_Income - cap_deduction - 1000000)*0.30 + 125000


print('your total tax amount for this year is :'+str(total_tax))




